.github/labels.yml

```diff
@@ -0,0 +1,91 @@
+- name: bug
+  description: Something isn't working
+  color: d73a4a
+- name: ci
+  description: CI/CD
+  color: dce775
+- name: build
+  description: build tools
+  color: dce775
+- name: dependencies
+  description: Pull requests that update a dependency file
+  color: dce775
+- name: documentation
+  description: Improvements or additions to documentation
+  color: 0075ca
+- name: enhancement
+  description: New feature or request
+  color: 64b5f6
+- name: feature
+  description: A new feature!
+  color: 26A763
+- name: good first issue
+  description: Good for newcomers
+  color: ffd54f
+- name: help wanted
+  description: Extra attention is needed
+  color: f043e1
+- name: performance
+  description: Performance topics
+  color: 64b5f6
+- name: question
+  description: Further information is requested
+  color: ba68c8
+- name: security
+  description: Security issue or vulnerability
+  color: e57373
+- name: breaking
+  description: Breaking change
+  color: f21d32
+
+# Meta
+- name: blocked
+  description: This is blocked
+  color: f7ae7e
+- name: duplicate
+  description: This issue or pull request already exists
+  color: cfd3d7
+- name: invalid
+  description: This doesn't seem right
+  color: ff088c
+- name: wontfix
+  description: This will not be worked on
+  color: cfd3d7
+- name: DO NOT MERGE
+  description: Not ready yet!
+  color: e56992
+
+# SDK Clients
+- name: android
+  description: Android client changes
+  color: 32DE84
+- name: jvm
+  description: JVM client changes
+  color: F58219
+- name: ingestion
+  description: Ingestion client changes
+  color: 180AF7
+
+# SDKs
+- name: core
+  description: Core SDK changes
+  color: 000080
+- name: auth
+  description: Auth SDK changes
+  color: 000080
+- name: crypto
+  description: Crypto SDK changes
+  color: 000080
+- name: securestore
+  description: Securestore SDK changes
+  color: 000080
+
+
+# Sample Apps
+- name: sample-android
+  description: SampleApp Android
+  color: 4514b7
+- name: sample-jvm
+  description: SampleApp JVM
+  color: 4514b7
```

Comment:
I am wondering, if we want to differentiate the domains by color, like for the clients...but it is in my eyes no blocker.